.github/workflows/stale.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale PR"
```

Comment:
you don't need this whole directory

Reply:
closed on [this commit](https://github.com/stethewwolf/flathub/commit/39a19a45598c846f928cbeeb49e63ca18d789ef6)